views/artist-search-results.hbs

```diff
@@ -0,0 +1,25 @@
+<body id="page2body">
+  <div class="page2container">
```

Comment:
I recommend not using numbers when you define class names or ids. You can use '-' or camel case when dealing with multiple words (i.e: page-container or pageToContainer)